src/util/time.h

```diff
@@ -71,6 +71,9 @@ int64_t GetTimeMillis();
 /** Returns the system time (not mockable) */
 int64_t GetTimeMicros();
 
+template <typename T>
+T GetSystemTime();
```

Comment:
276cfd2f5fd48d0556ccc4a83d350c0e9d45cc99: why is this file changed?

Reply:
Sorry, it was leftover... Fixed as of 6612412dfc